src/software/world/ball.h

```diff
@@ -91,6 +112,14 @@ class Ball final
      */
     Vector velocity() const;
 
+    /**
+     * Gets the previous velocities for each state stored in states_
+     *
+     * @return Vector containing the velocity history starting with the oldest available
+     * data at index 0
```

Comment:
Isn't the newest data at the "front" of the buffer which is index 0?

Reply:
The old class was using push_back so I used it and I think it loops from front to back so it'll be the oldest one